ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/nasm/tasks/main.yml

```diff
@@ -85,10 +85,19 @@
     - (nasm_installed.rc is defined) and ((nasm_installed.rc != 0 ) or (nasm_installed.rc == 0 and nasm.stdout is version_compare('2.13.03', operator='ne')) )
   tags: nasm
 
-- name: Running ./configure & make for nasm
-  shell: cd /tmp/nasm-2.13.03 && CC={{ CC }} ./configure -prefix=/usr/local && make install
+- name: Running ./configure & make for nasm ( Not Ubuntu 22+ )
+  shell: cd /tmp/nasm-2.13.03 && CC={{ CC }} && ./configure -prefix=/usr/local && make install
   when:
-    - (nasm_installed.rc is defined) and ((nasm_installed.rc != 0 ) or (nasm_installed.rc == 0 and nasm.stdout is version_compare('2.13.03', operator='ne')) )
+    - (ansible_distribution != "Ubuntu" or ansible_distribution == "Ubuntu" and ansible_distribution_major_version < "22") and (nasm_installed.rc is defined) and ((nasm_installed.rc != 0 ) or (nasm_installed.rc == 0 and nasm.stdout is version_compare('2.13.03', operator='ne')) )
+  tags: nasm
+
+- name: Running ./configure & make for nasm ( Ubuntu 22+ x64 )
```

Comment:
Hmmm so this will do nothing on Ubuntu 22.04 on non-x64?

Reply:
Yup, Im currently restricting it to where the problem has been seen.. I'll try and do a test on Arm64 Ubuntu 22 before merging.

Reply:
Ah, nasm is only installed on x64, so it doesn't impact ARM in this playbook. Im just checking a build on arm64 ubuntu, for the other half of this fix.

Reply:
I need to reflect the build change for Arm64 though...

Reply:
> Ah, nasm is only installed on x64

Fair point - I missed that ;-) Will approve now